blank_rust_project/contract/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
+cp target/wasm32-unknown-unknown/release/status_message.wasm ./res/
+#wasm-opt -Oz --output ./res/status_message.wasm ./res/status_message.wasm
```

Comment:
is this commented out on purpose?

Reply:
It's like this in near-bindgen examples as well, but I agree that we should not have commented out code. Removed.